src/edge_utils.c

```diff
@@ -2389,6 +2388,35 @@ static int edge_init_routes(n2n_edge_t *eee, n2n_route_t *routes, uint16_t num_r
   }
 #endif
 
+#ifdef WIN32
```

Comment:
Can you please cleanup this code so that each platform has a clean way of injecting rules?